main.py

```diff
@@ -64,6 +64,39 @@
 PUNISHMENT_INACTIVITY = 5
 
 
+def calculate_all_fitness(end_round_time, player_list, genomes_list):
+    if time.time() >= end_round_time or len(player_list) < 2:
+            genomes_list = list(map(calculate_fitness, player_list))
+            #for player_index, player in enumerate(player_list):
+            #    genomes_list[player_index].fitness = calculate_fitness(player)
+            print("Round time done")
+            for i in reversed(range(3)):
+                print(f"Starting next generation in {i+1}...", end="\r")
+                time.sleep(1)
+            return True
+    return False
+
+def correct_conflicting_movements(keys):

```

Comment:
Bias to specific move rather than discouraging the neural networks from choosing conflicting movements. Also, look at the RIGHT move... How is that boolean being evaluated?3

Reply:
Good to know I'm exactly the same at writing code outside of work as I am at it. Classic